tests/error/inout_errors/already_used.err

```diff
@@ -1,7 +1,12 @@
-Guppy compilation failed. Error in file $FILE:21
+Error: Linearity violation (at $FILE:21:8)
+   | 
+19 | def test(q: qubit @owned) -> None:
+20 |     use(q)
+21 |     foo(q)
+   |         ^ Variable `q` with linear type `qubit` cannot be borrowed
+   |           ...
+   | 
+20 |     use(q)
+   |         - since it was already consumed here
```

Comment:
"consumed" meaning enters another scope, and "moved" meaning belonging to a new name within same scope?

Reply:
Exactly!